packages/lit-dev-tools-esm/src/generate-js-samples.ts

```diff
@@ -229,10 +194,46 @@ const tsCompileOpts: InvokeTypeScriptOpts = {
   },
 };
 
+const nonTsFileGlobs = [
+  '**/*',
```

Comment:
Non blocking question. This was `'.'`. Are the two interchangeable?

Reply:
I found that chokidar understood `.`, but `fast-glob` didn't. Both understand `**/*` though.